src/stack_pr/cli.py

```diff
@@ -1310,8 +1322,31 @@ def create_argparser() -> argparse.ArgumentParser:
     return parser
 
 
+def load_config(config_file):
+    config = configparser.ConfigParser()
+    config["DEFAULT"] = {
+        "common": {
+            "verbose": False,
+            "hyperlinks": True,
+            "keep-body": False,
+            "draft": False,
+        },
+        "repo": {
+            "reviewer": None,
+            "remote": "origin",
+            "target": "main",
+        },
+    }
```

Comment:
Q: Aren't these defaults redundant with the ones above?

Reply:
Oh, good catch!